src/platform/user/profile/vet360/util/local-vet360.js

```diff
@@ -418,6 +418,30 @@ export default {
       1000,
     );
   },
+  addressValidationError() {
```

Comment:
Made another mock API call that mimics the error response we get from the API in staging.